scraping/src/commonJvmAndroid/kotlin/cz/lastaapps/scraping/AgataWallet.kt

```diff
@@ -0,0 +1,94 @@
+package cz.lastaapps.scraping
+
+import io.ktor.client.HttpClient
+import io.ktor.client.plugins.defaultRequest
+import io.ktor.client.request.forms.FormDataContent
+import io.ktor.client.request.get
+import io.ktor.client.request.header
+import io.ktor.client.request.post
+import io.ktor.client.request.setBody
+import io.ktor.client.statement.bodyAsText
+import io.ktor.http.Parameters
+import it.skrape.core.htmlDocument
+import java.net.URLDecoder
+
+actual class AgataWallet {
+    private val client: HttpClient = HttpClient {
+        // Disable redirects, because when I was testing it with Python I got into loop sometimes
+        // Also it has to extract some cookies from some of the requests, so redirects manually handled
+        followRedirects = false
+
+        defaultRequest {
+            header("User-Agent", "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36")
```

Comment:
Je to opravdu potřeba? Nebo to je ochrana před tím, aby to neblikli eventuelně.

Plus kdyžtak toto by mělo stačit: https://ktor.io/docs/user-agent.html#install_plugin